github-approve-label-workflow/github-approve-label-workflow.py

```diff
@@ -60,6 +60,13 @@ def movelabeldonetoready(issue):
     res = requests.post(url, data=json.dumps(newlabel), headers=headers)
     if (res.status_code != 200):
         print("Error adding label", res.status_code, res.content)
+        return
+    newcomment = {"body":"This issue is ready to merge"}
```

Comment:
s/issue/pull request/ ????

Or just say "This is ready to merge" ?

Reply:
everything is an issue, but ok :)

Reply:
The URL I clicked through on has "pull" not "issues" :)